_posts/2019-03-07-strimzi-0.11.0-release.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: post
+title:  "Strimzi 0.11.0 and 0.11.1 released!"
+date: 2018-03-07
+author: paolo_patierno
+---
+
+We are delighted to announce the new Strimzi 0.11.0 and 0.11.1 releases with some new features!
+
+<!--more-->
+
+# JBOD storage support for Kafka brokers
+
+Other than supporting the ephemeral and persistent storage, now Strimzi supports JBOD (Just a Bunch of Disks) storage for Kafka brokers (but not for Zookeeper nodes).
+
+The JBOD data storage configuration provides a way for having multiple disks combined to create a single logical disk, or volume.
```

Comment:
There's not combining into a single logical volume going on in the accepted sense (e.g. RAID, or the various linuxy ways of doing this). It's just a bunch of (unrelated) disks which the broker is putting the partition data on.

```suggestion
The JBOD data storage configuration provides a way for Kafka brokers to make use of multiple disks.
```

Reply:
We have to change the documentation as well then ;) I will open a PR for that.

Reply:
@ppatierno Please sync with Paul for the docs ... I raised it today duirng the release notes review and he mentioned it is in the docs like this as well.